transactions/src/test/java/org/ehcache/transactions/xa/internal/XAStoreTest.java

```diff
@@ -1488,6 +1490,52 @@ public void testBulkComputeIfAbsent() throws Exception {
   }
 
   @Test
+  public void testCustomEvictionVeto() throws Exception {
```

Comment:
Are we ruling this test sufficient?  It obviously verifies that the custom veto is called... it doesn't actually verify that there is any effect on the eviction process however (of course whether that could be verified is another question).